airbyte-integrations/connectors/source-google-analytics-data-api/unit_tests/test_source.py

```diff
@@ -43,7 +43,7 @@ def patch_base_class():
 @pytest.fixture
 def config():
     return {
-        "property_id": "108176369",
+        "property_ids": ["108176369"],
```

Comment:
can we add unit test with old config and try test backward compatibility in that way